zqd/handlers_test.go

```diff
@@ -371,9 +371,10 @@ func TestPostZngLogs(t *testing.T) {
 	status := payloads[len(payloads)-2].(*api.LogPostStatus)
 	span := &nano.Span{Ts: 1e9, Dur: 1e9 + 1}
 	require.Equal(t, &api.LogPostStatus{
-		Type: "LogPostStatus",
-		Span: span,
-		Size: 80,
+		Type:         "LogPostStatus",
+		Span:         span,
+		Size:         80,
+		LogTotalSize: 148,
```

Comment:
Should we also have `LogReadSize: 148` here as well?

Reply:
I looked at doing this, and thought it would be complex, since the value for LogReadSize comes from the driver.Ouput stat call, but I just realized how to do it.